examples/coreos/README.md

```diff
@@ -0,0 +1,33 @@
+# CoreOS multi-machine example setup
+
+### Requirements
+* Linux ~> 4.14
```

Comment:
this requirements here are for the `kvm-host`?

Reply:
e.g i would state `linux-kernel` since `Linux` is generic word imho.

Reply:
That is indeed the linux-kernel requirement. It should be a general requirement for this provider if you create more than one machine with graphics {} at a time it will cause issues.
I mentioned it on the gitter channel but that seems to be mostly dead:
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=863266
https://bugzilla.redhat.com/show_bug.cgi?id=1432684